Source/CBL_ForestDBStorage.mm

```diff
@@ -1063,6 +1079,11 @@ - (CBLStatus) forceInsert: (CBL_Revision*)inRev
 
     if (change)
         [_delegate databaseStorageChanged: change];
+
+    if (CBLStatusIsError(status)) {
+        if (outError && !*outError)
```

Comment:
!*outError check is required here to not overriding the validation error from line 1060.